mdjs-layout/src/MdjsLayout.js

```diff
@@ -1,7 +1,11 @@
 import { html, LitElement } from 'lit';
 import { unsafeHTML } from 'lit/directives/unsafe-html.js';
-import '../mdjs-layout-controls/mdjs-layout-controls.js';
-import './mdjs-styles.css';
+import '../../mdjs-layout-controls/mdjs-layout-controls.js';
+import styles from './mdjs-styles.css.js';
+
+const sheet = new CSSStyleSheet();
```

Comment:
You need a polyfill for being able to use this I think

Reply:
Added polyfill

Reply:
I checked also on firefox, works nicely there :)